workspaces/tests/account.rs

```diff
@@ -32,6 +33,18 @@ async fn test_subaccount_creation() -> anyhow::Result<()> {
         Some(&Value::String(sub.id().to_string()))
     );
 
+    let res = worker
+        .delete_account(sub.id(), sub.signer(), &worker.info().root_id)
+        .await?;
+    assert!(res.is_success());
+
+    let res = worker
+        .delete_account(account.id(), account.signer(), &worker.info().root_id)
+        .await?;
+    assert!(res.is_success());
```

Comment:
Failing tests must clean up the accounts as well to avoid draining the faucet account.

Just for context, the previous comment on this topic was here: https://github.com/near/near-workspaces-rs/pull/369/files#r1713257285, please, either fix the drainage or go back to sandbox.

Reply:
Returned to the sandbox. The main idea of ​​staying on testnet was that there are no tests on testnet